docs/azure/get-started-with-azure/guardrails.md

```diff
@@ -0,0 +1,220 @@
+4.  **Data Protection:** How your data is protected at rest and in transit.
+5.  **Cost Optimization:** Preventing unnecessary costs.
+6.  **Identity & Access Management:** Controlling who can access resources.
+7.  **Specific Service Guardrails:** Restrictions related to specific Azure services.
+8.  **Monitoring:** Diagnostic data collection and automated security checks.
+9.  **Tagging:** Automatic application of tags.
+
+## Resource deployment restrictions
+
+*   **Allowed Locations (Regions):** You can only deploy resources in Canada Central and Canada East. If you try to deploy to a region not on the approved list, the deployment will be **blocked**. Note that networking is only provided in Canada Central.
+*   **Resource Group Locations:** The location (region) of a resource group *must* match the location of the resources within it.
+*   **Denied Resource Types:** There is a "Deny" policy for specific resource types that are *not* allowed within the landing zone, preventing their creation.
+
+## Networking restrictions
+
+### General network security
+*   **No Public IPs (Generally):** You *cannot* create public IP addresses directly. This is a crucial security measure to minimize our public attack surface. Services should typically be accessed through private endpoints or other secure means.
+*   **No Public IP on NICs:** Network interfaces *cannot* have a public IP address associated.
+*   **No Management Port Access from Internet:** NSG rules allowing inbound access from the "Internet" source on common management ports (SSH/RDP - ports 22 and 3389) are *blocked*. Access to these ports is only allowed through secure, managed pathways (e.g., Azure Bastion, Just-In-Time VM Access).
+
+### Subnet configuration requirements
+*   **Subnets Require Network Security Groups (NSGs):** Every subnet *must* have a Network Security Group (NSG) associated with it. You *cannot* create a subnet without an NSG. NSGs act like mini-firewalls for your subnets, controlling inbound and outbound traffic.
+*   **Subnets Require Private Endpoint Network Policies**: Subnets created must have the private endpoint network policy enabled.
+*   **Subnets Require User Defined Routes (UDRs):** Every subnet (except specific exceptions like `AzureBastionSubnet` and gateway subnets) *must* have a User-Defined Route (UDR) table associated. This is used to force traffic through a central firewall (e.g., Azure Firewall) in the hub VNet.
+*   **No Service Endpoints:** Service Endpoints are *denied*. The preferred approach is to use Private Endpoints for secure access to PaaS services.
+
+### VNet management
+*   **No VNet Creation:** New VNets *cannot* be created in the landing zone.
+*   **No DNS Changes to VNet:** VNet DNS settings are enforced and cannot be changed. This is configured at the landing zone level.
+*   **Azure Firewall Policy Required:** If an Azure Firewall is deployed, it *must* have an associated Firewall Policy.
```

Comment:
Do the end users need to know about our enforced configuration of Azure Firewall? I kind of would think they don't care.

Reply:
I wasn't sure if maybe they could deploy their own firewall if they wanted to.

Reply:
Hmm, not sure. It might be in the protected resources list. I'll check.

Reply:
let's jusdt remove it for now. It is unlikely that teams wiill use it and it adds confusion